### PR TITLE
Convert DrawerNavGroup to live code examples

### PR DESCRIPTION
**Pull request by priyankakmEaton**
Fixes #5790

#### Changes proposed in this Pull Request:

 -Convert images to live RN code (using react-native-web)

-Remove unused image assets

-Convert markdown code blocks to PrismJS code blocks

-Use PrismJS to highlight targeted lines in the code example blocks

-Include buttons to copy to clipboard and view full github source

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Comment by JeffGreiner-eaton** [comment]
looks like we need to use theme.colors for itemFontColor and titleColor...etc

Drawer nav group in dark theme

image

File: docs/src/componentDocs/DrawerNavGroup/examples/SpacingNavGroups.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import {CodeBlock, CodeBlockActionButtonRow} from '../../../shared';
+import {SpacingNavGroupsExample} from './SpacingNavGroupsExample';
+
+const codeSnippet = `<Drawer style={{width: 250, margin: 'auto'}}>
+    <DrawerBody>
+        <DrawerNavGroup title="Group 1" hidePadding titleDivider={false}>
+            <DrawerNavItem title="Item 1" itemID="1" />
+            <DrawerNavItem title="Item 2" itemID="2" />
+        </DrawerNavGroup>
+        <Spacer height={30} style={{flexBasis: 'auto'}} />
```
Comment (JeffGreiner-eaton) [inline docs/src/componentDocs/DrawerNavGroup/examples/SpacingNavGroups.tsx:12]: you can remove style={{flexBasis: 'auto'}} from snippet, this is just a fix for a bug with RNWeb

Comment (priyankakmEaton) [reply docs/src/componentDocs/DrawerNavGroup/examples/SpacingNavGroups.tsx:12]: Spacer height is not reflecting without flexBasis: auto, hence added it.

File: docs/src/componentDocs/DrawerNavGroup/examples/DrawerNavGroupComposition.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                    itemID: '4',
+                },
+            ]}
+        />
+    </DrawerBody>
+</Drawer>`;
+export const DrawerNavGroupComposition = (): JSX.Element => (
+  <Box>
+    <DrawerNavGroupCompositionExample />
+    <CodeBlock code={codeSnippet} language="jsx" dataLine="3-21" />
```
Comment (surajeaton) [inline docs/src/componentDocs/DrawerNavGroup/examples/DrawerNavGroupComposition.tsx:33]: `dataLine` should be feom 3-22

**Review by surajeaton — APPROVED** [review]

**Pull request merged**